src/factions/AdeptaSororitas/faction.json

```diff
@@ -0,0 +1,6 @@
+{
+    "Id": "ADS",
```

Comment:
is AS already taken? A lot of the other faction IDs are two letters.

Reply:
Yeah it was taken for Eldar, ran into FK issues when I used AS so looked it up.